src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/prison/api/OffenderRestrictionDto.kt

```diff
@@ -21,7 +21,7 @@ data class OffenderRestrictionDto(
   val startDate: LocalDate,
 
   @Schema(description = "Date restriction applies to, or indefinitely if null", example = "1980-01-01")
-  val expiryDate: LocalDate,
+  val expiryDate: LocalDate? = null,
 
```

Comment:
Scary bit to this is it been here since 22/02/2023

Reply:
yes, I think this API call wasn't used.